app/views/samples/index.haml

```diff
@@ -43,7 +43,7 @@
                 %td
                   =check_box_tag 'sample_ids[]', sample.id, false, { id: "sample_ids.#{sample.id}" }
                   %label.row{for: "sample_ids.#{sample.id}"}
-                %td{id: 'sample_uuid'}= short_uuid_with_title sample.uuid
+                %td{id: 'sample_uuid', value: sample.uuid}= short_uuid_with_title sample.uuid
```

Comment:
Rather than inventing a custom `value` attribute, this should use a `data-*` attribute. `data-` is essentially a namespace for custom attributes.
```suggestion
                %td{id: 'sample_uuid', data: {uuid: sample.uuid}}= short_uuid_with_title sample.uuid
```